prime-router/src/test/resources/fhirengine/engine/bundle_multiple_observations_filtered_only_94558_5.fhir

```diff
@@ -0,0 +1,1680 @@
+        "meta": {
+          "extension": [
+            {
+              "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/source-processing-id",
+              "valueCodeableConcept": {
+                "coding": [
+                  {
+                    "system": "https://terminology.hl7.org/3.1.0/CodeSystem-v2-0103.html",
+                    "code": "P"
+                  }
+                ]
+              }
+            },
+            {
+              "url": "http://ibm.com/fhir/cdm/StructureDefinition/source-record-id",
+              "valueId": "1234d1d1-95fe-462c-8ac6-46728dba581c"
+            },
+            {
+              "url": "http://ibm.com/fhir/cdm/StructureDefinition/source-data-model-version",
+              "valueString": "2.5.1"
+            }
+          ]
+        },
+        "name": "Winchester House",
+        "contact": [
+          {
+            "purpose": {
+              "coding": [
+                {
+                  "system": "http://terminology.hl7.org/CodeSystem/contactentity-type",
+                  "code": "ADMIN",
+                  "display": "Administrative"
+                }
+              ],
+              "text": "Organization Medical Director"
+            }
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "Device/1667861767911610000.bd8aa647-0013-41c3-9d51-e05fa01876ea",
+      "resource": {
+        "resourceType": "Device",
+        "id": "1667861767911610000.bd8aa647-0013-41c3-9d51-e05fa01876ea",
+        "meta": {
+          "extension": [
+            {
+              "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/source-processing-id",
+              "valueCodeableConcept": {
+                "coding": [
+                  {
+                    "system": "https://terminology.hl7.org/3.1.0/CodeSystem-v2-0103.html",
+                    "code": "P"
+                  }
+                ]
+              }
+            },
+            {
+              "url": "http://ibm.com/fhir/cdm/StructureDefinition/source-record-id",
+              "valueId": "1234d1d1-95fe-462c-8ac6-46728dba581c"
+            },
+            {
+              "url": "http://ibm.com/fhir/cdm/StructureDefinition/source-data-model-version",
+              "valueString": "2.5.1"
+            }
+          ]
+        },
+        "identifier": [
+          {
+            "type": {
+              "coding": [
+                {
+                  "system": "http://terminology.hl7.org/NamingSystem/uri",
+                  "code": "2.16.840.1.114222.4.1.237821",
+                  "display": "CDC PRIME - Atlanta,"
+                }
+              ]
+            }
+          }
+        ],
+        "deviceName": [
+          {
+            "name": "CDC PRIME - Atlanta,",
+            "type": "user-friendly-name"
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "Observation/f3f94c27-e225-4aac-b6f5-2750f45dac4f",
```

Comment:
@arnejduranovic @mkalish making the change you (@mkalish ) requested results in four test failures. I suspect those other tests are expecting the same value this test was formerly expecting before I made the requested change. 

Do you want me to revert the change here or figure out why those other tests are broken as a result? 

![Screenshot 2024-06-06 at 10 12 54 PM](https://github.com/CDCgov/prime-reportstream/assets/165852085/259424f8-94c4-48e0-ad44-2f03cf14677b)

Reply:
I haven't looked but I'm pretty sure those tests are failing because they are doing the same thing with that file my tests are - looking for a specific id tied to that Observation element. 

And my feeling is it's out of scope to make these changes here given other surgery in other unrelated parts of the system are breaking as a result. 

If @mkalish wants to ticket the work I'm happy to pick it up and take care of it. I agree that we shouldn't try to overload the scope this PR any further than it's already been overloaded. I'll revert the change and re-request review